src/test_xor128.rs

```diff
@@ -9,7 +9,7 @@ use rayon::iter::ParallelIterator;
 use std::iter::FromIterator;
 use std::fmt;
 use std::hash::Hasher;
-use store::{Store, DiskStore, VecStore, LevelCacheStore};
```

Comment:
lets unexport the `LevelCacheStore` for now, or at least put a big warning on it to make clear that it does not work at the moment